storage/swift_storage.py

```diff
@@ -121,17 +94,12 @@ def _download_object_to_file(
             self, container: str, object_name: str, out_file: BinaryIO) -> None:
         connection = self.get_connection()
 
-        def get_object() -> None:
-            out_file.seek(0)
-            resp_headers, object_contents = connection.get_object(
-                container, object_name, resp_chunk_size=_LARGE_CHUNK)
-            # may need to truncate file if retrying...
-            for object_content in object_contents:
-                out_file.write(object_content)
-
-        retry_swift_operation(
-            f"Failed to retrieve Swift object {object_name} from container {container}",
-            get_object)
+        out_file.seek(0)
+        resp_headers, object_contents = connection.get_object(
+            container, object_name, resp_chunk_size=_LARGE_CHUNK)
+        # may need to truncate file if retrying...
```

Comment:
Very minor, and may have been introduced before you got to it, but I believe this comment is talking about the `out_file.seek(0)` line.